papaya-integration-tests/src/test/java/pl/edu/agh/papaya/tests/projects/ProjectsFixture.java

```diff
@@ -33,4 +34,14 @@ public ProjectDto getProjectById(Long id) throws ApiException {
     public List<ProjectDto> getProjects() throws ApiException {
         return projectsApi.getProjects();
     }
+
+    public void addPeterToProject(long projectId) throws ApiException {
```

Comment:
Trzy razy czytałam nazwę tej metody, zanim ogarnęłam, że to są testy :stuck_out_tongue: